src/font.js

```diff
@@ -29,12 +29,21 @@ define(['propertyParser'], function (propertyParser) {
             if (config.isBuild) {
                 onLoad(null); //avoid errors on the optimizer
             } else {
-                var data = parseName(name);
+                var data = parseName(name),
+                    fontLoaderVersion = '1.5'; // Available versions listed at
```

Comment:
I would suggest to use webfontVersion instead of fontLoaderVersion perhaps?
It would look more clear to me, what do you think?